monai/bundle/scripts.py

```diff
@@ -219,6 +224,47 @@ def _download_from_ngc(
     extractall(filepath=filepath, output_dir=extract_path, has_base=True)
 
 
+def _download_from_nvstaging(
```

Comment:
`ngc_private`

Reply:
Thanks, updated it in the latest commit.